source/options-storage.ts

```diff
@@ -20,6 +20,7 @@ const defaults = Object.assign({
 const migrations = [
 	featureWasRenamed('sticky-conversation-sidebar', 'sticky-sidebar'), // Merged in September
```

Comment:
```suggestion
```
W can drop this line

Reply:
It’s still October 😃 include it in your November lint